jitar/src/core/Implementation.ts

```diff
@@ -1,5 +1,6 @@
 
 import MissingParameterValue from './errors/MissingParameterValue.js';
+import UnknownParameter from './errors/UnknownParameter'
```

Comment:
In the core and the runtime module we add a '.js' postfix to the import files. We need this in order to make the same code run in the web browser. This isn't very developer friendly, but we haven't found a better solution yet. Ideas are welcome!